app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -96,13 +97,24 @@ open class DuckDuckGoApplication : HasDaggerInjector, MultiProcessApplication()
     }
 
     override fun onSecondaryProcessCreate(shortProcessName: String) {
-        AndroidLogcatLogger.installOnDebuggableApp(this, minPriority = LogPriority.VERBOSE)
+        configureLogging()
```

Comment:
Ah so we are still configuring Timber then.

Reply:
This is really for the logs in this `onSecondaryProcessCreate`. Maybe I just remove it ?

Reply:
Ah I see. I think it is better not to have this because it will enforce more the usage of  logcat in VPNProcess. Also the logs are just debug logs anyway.

Reply:
Makes sense, will remove Timber entirely then.